src/tactic/sort.lean

```diff
@@ -0,0 +1,153 @@
+/-
+Copyright (c) 2022 Arthur Paulino, Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Arthur Paulino, Damiano Testa
+-/
+import data.polynomial.degree.definitions
+
+/-!  # A tactic for sorting sums  -/
+
+namespace tactic
+
+/--  Given an expression `ei` and a list `l : list expr`, returns a `tactic expr` that is
+`ei + sum_of_list l`. -/
+meta def build_sum (ei : expr) (l : list expr) : tactic expr :=
+l.reverse.mfoldr (λ e1 e2, mk_app `has_add.add [e2, e1]) ei
```

Comment:
Is there an mfoldl?

Reply:
`mfoldl` exists and works better, there, thanks!